cmds/pflask/pflask.go

```diff
@@ -19,6 +19,9 @@ import (
 	"time"
 
 	"unsafe"
+
+	// "github.com/u-root/u-root/pkg/termios"
```

Comment:
If this works feel free to remove the commented out package.

Reply:
Ok, will check. I only rudimentarily tested pflask with this PR, so I will check again whether it still works.